lib/property_sets/active_record_extension.rb

```diff
@@ -111,8 +111,8 @@ def disable(arg)
         send("#{arg}=", "0")
       end
 
-      def build_default(arg)
-        build(:name => arg.to_s, :value => raw_default(arg))
+      def build_default(arg, owner=nil)
```

Comment:
I think it can't, I was being a bit defensive but I can remove the ```= nil``` here and on the other method